lib/ansible/modules/notification/cisco_spark.py

```diff
@@ -0,0 +1,198 @@
+    personal_token: "{{ token }}"
+    message: "Cisco Spark Ansible Module - Room Message in Text"
+
+- name: Cisco Spark - Text Message by an Individuals ID
+  cisco_spark:
+    recipient_type: toPersonId
+    recipient_id: "{{ person_id}}"
+    message_type: text
+    personal_token: "{{ token }}"
+    message: "Cisco Spark Ansible Module - Text Message to Individual by ID"
+
+- name: Cisco Spark - Text Message by an Individuals E-Mail Address
+  cisco_spark:
+    recipient_type: toPersonEmail
+    recipient_id: "{{ person_email }}"
+    message_type: text
+    personal_token: "{{ token }}"
+    message: "Cisco Spark Ansible Module - Text Message to Individual by E-Mail"
+
+"""
+
+RETURN = """
+status_code:
+  description:
+    - The Response Code returned by the Spark API
+    - Full Responsde Code explanations can be found at https://developer.ciscospark.com/endpoint-messages-post.html
+  returned: always
+  type: int
+  sample: 200
+
+message:
+    description:
+      - The Response Message returned by the Spark API
+      - Full Responsde Code explanations can be found at https://developer.ciscospark.com/endpoint-messages-post.html
+    returned: always
+    type: string
+    sample: OK (585 bytes)
+"""
+
+
+def spark_message(module):
+    """ Send a message to a Cisco Spark Room or Individual """
+
+    # Ansible Specific Variables
+    results = {}
+    ansible = module.params
+
+    # Cisco Spark API Variables
+    url = "https://api.ciscospark.com/v1/messages"
+
+    headers = {
+        'Authorization': 'Bearer {}'.format(ansible['personal_token']),
+        'content-type': 'application/json'
+    }
+
+    # By Room ID
+    payload = {
+        ansible['recipient_type']: ansible['recipient_id'],
+        ansible['message_type']: ansible['message']
+    }
+
+    payload = module.jsonify(payload)
+
+    response, info = fetch_url(module, url, data=payload, headers=headers)
+
+    status_code = info['status']
+    message = info['msg']
+
+    # Module will fail if the response is not 200
+    if status_code != 200:
+        results['failed'] = True
+        results['status_code'] = status_code
+        results['message'] = message
+
+    else:
+        results['failed'] = False
+        results['status_code'] = status_code
+        results['message'] = message
+
+    return results
+
+
+def main():
+    '''Ansible main. '''
+    module = AnsibleModule(
+        argument_spec=dict(
+            recipient_type=dict(required=True, choices=[
+                'roomId', 'toPersonEmail', 'toPersonId']),
+            recipient_id=dict(required=True, no_log=True),
+            message_type=dict(required=False, default=['text'], aliases=[
+                              'type'], choices=['text', 'markdown']),
+            personal_token=dict(required=True, no_log=True, aliases=['token']),
+            message=dict(required=True)
+
+        ),
+
+        supports_check_mode=True
+    )
+
+    # Add Check Mode Support
+    if module.check_mode:
```

Comment:
Unfortunately, no. The API expects a payload and if none are provide it returns a 400 error status code (The request was invalid or cannot be otherwise served.)